account_payment_partner/models/account_move.py

```diff
@@ -57,14 +58,26 @@ def _compute_payment_mode(self):
         for move in self:
             move.payment_mode_id = False
             if move.partner_id:
+                partner = move.with_context(force_company=move.company_id.id).partner_id
                 if move.type == "in_invoice":
-                    move.payment_mode_id = move.with_context(
-                        force_company=move.company_id.id
-                    ).partner_id.supplier_payment_mode_id
+                    move.payment_mode_id = partner.supplier_payment_mode_id
```

Comment:
Why do you remove force_company?

Reply:
It's already in https://github.com/OCA/bank-payment/pull/810/files#diff-560b5e454acd270e25223d56b85967eaec36fe9a08eac999124990650aa32f6cR61

Reply:
Ups, I see, sorry